### ISSUE TITLE
Fontawesome icons not displayed on Elgg 1.X

### ISSUE THREAD

**Issue by iionly** [hypeJunction/hypeWall#68]
Tested on Elgg 1.9 and 1.0 with hypeWall 4.3.2, hypeApps 4.3.3 and hypeFilestore 3.0.0 and 3.1.1 (on Elgg 1.10). The fontawsome icons in the hypeWall menu on the activity page are not displayed.

On Elgg 2.0 (beta) it seems to work fine (displaying of icons... not tested anything else).

**Comment by hypeJunction** [comment]
I have removed the font from individual plugins, so you can either add it to your theme, or use ColdTrick's fontawesome plugin.

**Comment by iionly** [comment]
Maybe a suggests in manifest to use the fontawesome plugin would be helpful or some remark in the Readme for users to know what to do. I hadn't realized that the fontawesome support within hypeWall was removed when testing compatibility of my hypeWall-izap videos pluigin with the recent Elgg versions. For me it just looked like the 4.X versions not fully working on Elgg 1.X (i.e. no icons showing up anymore). That's also the reason I recommened the use of the latest 3.X version of hypeWall for Elgg 1.9 - 1.12 and the latest 2.X version for Elgg 1.8 when making a new release of my plugin. For Elgg 2.X using fontawesome is supported by core anyway. Though I'm not sure if replacing all the icons within Elgg 1.X with fontawesome icons just to get the icons used by one 3rd party plugin might be problematic with some other 3rd party plugins. But that's not for me to decide.
